utils/api.js

```diff
@@ -0,0 +1,8 @@
+import axios from "axios";
+export default axios.create({
+    baseURL:"https://elite-bn-be.herokuapp.com/",
+    headers: {
+        'Authorization': 'Bearer ' + localStorage.getItem('token')
+      }
```

Comment:
define the token instead.
token = localStorage.getItem('token');
then concat it the variable with `Bearer` instead

Reply:
done